### PR TITLE
Js 13 create reusable components

### PR DESCRIPTION
**Pull request by Alekseiv25**
Add:  Primary, secondary buttons (active/inactive) , Title components, paragraph component, checkboxes
[Task link](https://jslab.atlassian.net/browse/JS-13?atlOrigin=eyJpIjoiMzAzYmIxMTExNWNjNDRiOWEzMDljZjk5NWRmMWUyYmUiLCJwIjoiaiJ9)
mobile
desktop

### CODE REVIEW

File: src/components/Paragraph/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    lineHeight: theme.lineHeights.paragraph.m,
+  },
+  paragraphL: {
+    fontSize: theme.fontSizes.paragraph.l,
+    lineHeight: theme.lineHeights.paragraph.l,
+  },
+  paragraphS: {
+    fontSize: theme.fontSizes.paragraph.s,
+    lineHeight: theme.lineHeights.paragraph.s,
+  },
```
Comment (alexandragul) [inline src/components/Paragraph/index.tsx:28]: There is no need in font sizes for header, label and paragraph separately. Please remove them, it's too complicated

File: src/styles/Theme.tsx [hunk 2]
```diff
@@ -42,52 +42,52 @@ export const theme: ITheme = {
...
-      m: 14,
-      s: 12,
-      xs: 11,
+      xl: '18px',
+      l: '16px',
+      m: '14px',
+      s: '12px',
+      xs: '11px',
     },
   },
```
Comment (alexandragul) [inline src/styles/Theme.tsx:67]: Please follow this example 

```
fontSizes: [
  '0.8125rem', // 0 13px
  '0.8750rem', // 1 14px
  '1rem', // 2 16px
  '1.125rem', // 3 18px
  '1.25rem', // 4 20px
  '1.5rem', // 5 24px
  '1.75rem', // 6 28px
  '1.875rem', // 7 30px
  '2rem', // 8 32px
  '2.375rem', // 9 38px
  '3.125rem', // 10 50px
]
```

File: src/types/theme.ts [hunk 3]
```diff
@@ -92,3 +92,7 @@ interface ITheme {
   };
 }
 export default ITheme;
+
+export interface StyledProps {
+  theme: ITheme;
+}
```
Comment (alexandragul) [inline src/types/theme.ts:98]: I believe it'd work the same way, you don't need to declare massive `ITheme`

```suggestion
export interface StyledProps {
  theme: typeof theme;
}
```

**Review by gribanovartem — COMMENTED** [review]
Please add screenshots (web and mobile)

File: src/components/Title/index.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+function Title(props: ITitleProps) {
+  const { children, variant, ...space } = props;
+  return (
+    <StyledTitle variant={variant} {...space}>
+      {children}
+    </StyledTitle>
+  );
+}
+
+export default Title;
```
Comment (gribanovartem) [inline src/components/Title/index.tsx:46]: in this case all titles will be h3. There should be ability to add h1, h2, ...

**Review by gribanovartem — APPROVED** [review]

**Pull request merged**